renovate.json5

```diff
@@ -74,6 +74,21 @@
         labels: ['Feature:Vega', 'Team:VisEditors'],
         enabled: true,
       },
+      {
+        groupName: 'platform security modules',
+        packageNames: [
+          'broadcast-channel', 
+          'jsonwebtoken', '@types/jsonwebtoken', 
+          'node-forge', '@types/node-forge', 
+          'require-in-the-middle', 
+          'tough-cookie', '@types/tough-cookie',
+          'xml-crypto', '@types/xml-crypto'
+        ],
+        reviewers: ['team:kibana-security'],
+        matchBaseBranches: ['master', '7.x'],
```

Comment:
Are we going to get rid of the 7.x branch when we cut 7.16? (I ask because there is currently no 6.x branch)

Reply:
> Are we going to get rid of the 7.x branch when we cut 7.16? (I ask because there is currently no 6.x branch)

Possibly/probably? Do you think we should just target `master` instead?

Reply:
We're also getting rid of master in favor of main 😅 perhaps we should just leave it like this and try to remember to change it later